src/components/QuranReader/TranslationView/TranslationViewCell.tsx

```diff
@@ -112,7 +112,7 @@ const TranslationViewCell: React.FC<TranslationViewCellProps> = ({
           </div>
         </div>
       </div>
-      <Separator />
+      <Separator isTranslationView />
```

Comment:
Since `Separator` is a general component and not necessarily related to the QuranReader or Translation view, we shouldn't have this param. What we can do is to add an optional `className` prop that we can pass the value of the className we want to be applied to the Seperator e.g. <Seperator className={styles.faded} and we will consume the prop `className` inside the `Seperator` component. Something like this:

```
<SeparatorPrimitive.Separator
    orientation={isVertical ? 'vertical' : 'horizontal'}
    className={classNames(styles.base,{[className]: className})}
  />
  ```
  
  You can check `Skeleton` component for reference.